### PR TITLE
Dashboard README update

### PR DESCRIPTION
**Pull request by aga11313**
- add image resources
- update deployment instructions
- align features descriptions with current state of the codebase

TODO:
- add a gif for the new Hub view (@BastinJafari is working on this)

### CODE REVIEW

**Comment by lusloher** [comment]
I added some more changes :) but really nice

File: README.md [hunk 1]
```diff
@@ -10,97 +10,151 @@
...
+npm run build-dashboard
+```
+- Run dashboard:
+```bash
+node dashboard
+```
+- Dashboard should now be served on `http://localhost:3030` by default
 
-## Contributing
+# Contributing
```
Comment (FionnD) [inline README.md:157]: Can you add the https://github.com/jina-ai/jina/#community  + https://github.com/jina-ai/jina/#open-governance and https://github.com/jina-ai/jina/#join-us section from the main repo please? Just copy and paste!

**Review by FionnD — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -10,97 +10,151 @@
...
 <img src=".github/.README_images/2859cc17.png?raw=true" alt="log server settings" width="80%">
 </p>
 
-You should now see the log stream and flow visualization.
-
-## Self-host Dashboard
+ - You should now be able to see the log stream and flow visualization.
 
-One can self-host a dashboard locally.
+# Self-host Dashboard
```
Comment (FionnD) [inline README.md:114]: Self-host**ed** (and in the next line)

**Review by FionnD — APPROVED** [review]

**Review by lusloher — APPROVED** [review]
just the one comment on Daemon and the first graphic still features some windows that are not up to date but other than that looks good

**Pull request merged**